Venue announce

miketraverso opened the pull request.

miketraverso committed:
Venue announcement blog post. Updated map on main page.

miketraverso committed:
Updates to sponsors.

miketraverso committed:
Allow three blog posts on front page.

miketraverso committed:
Correct sponsor type.

miketraverso committed:
reduce partner image sizes.

miketraverso committed:
more image reduction.

Splaktar commented on app/elements/blocks/logos-block/logos-block.html:
```diff
@@ -66,7 +66,7 @@ <h5 class="logos-block-title text-center">[[block.title]]</h5>
               <a class="logo-item" href="[[logo.url]]" title="[[logo.name]]" target="_blank">
                 <iron-image class="logo-img" src="[[logo.logoUrl]]" height$="{{logo.height}}"
-                            width$="{{logo.width}}" preload fade sizing="contain"></iron-image>
+                            width$="{{logo.width}}" fade sizing="contain"></iron-image>
```
Why was this change needed?

Splaktar committed:
update(blog,partners): add vaadin as a partner. enhance venue announcement blog post.

update tweet data
adjust sponsor logo sizes so that higher sponsorship has larger logos
add brief abstract for venue blog post as it shows up on home page
